pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1275,72 +980,6 @@ func setupManagerWithGCEnabled(stopCh chan struct{}, objs ...runtime.Object) (ma
 	return manager, fakePodControl, informers
 }
 
-func TestDoNotPatchPodWithOtherControlRef(t *testing.T) {
```

Comment:
This should stay a unit test (it checks that we do nothing) but refactored.

Reply:
noted.